docs/services/endlessh.md

```diff
@@ -0,0 +1,70 @@
+# Endlessh
+
+[Endlessh-go](https://github.com/shizunge/endlessh-go) is a A golang implementation of endlessh, a ssh trapit. Installing it is powered by the [mother-of-all-self-hosting/ansible-role-endlessh](https://github.com/mother-of-all-self-hosting/ansible-role-endlessh) Ansible role.
+
+## Dependencies
+
+This service requires the following other services:
+
+- (optionally) [Traefik](traefik.md) - a reverse-proxy server for exposing endlessh publicly
+- (optionally) [Prometheus](./prometheus.md) - a database for storing metrics
+- (optionally) [Grafana](./grafana.md) - a web UI that can query the prometheus datasource (connection) and display the logs
+
+## Installing
+
+To configure and install endlessh on your own server(s), you should use a playbook like [Mother of all self-hosting](https://github.com/mother-of-all-self-hosting/mash-playbook) or write your own.
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# endlessh                                                                 #
+#                                                                      #
+########################################################################
+
+endlessh_enabled: true
```

Comment:
Maybe we should mention the `endlessh_container_host_bind_port: 22` default and the fact that endlessh will try to bind to port 22 on all network interfaces by default.

This may be somewhat obvious, but it's probably good to mention it explicitly.